Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -197,14 +199,14 @@ def search_user(query: str, max_results: str = '50'):
     Returns:
         List of users.
     """
-    url = f"rest/api/latest/users/search?query={query}&maxResults={max_results}"
+    url = f"rest/api/latest/user/search?query={query}&maxResults={max_results}"
     res = jira_req('GET', url, resp_type='json')
     return res
 
 
 def get_account_id_from_attribute(attribute: str, max_results: str = '50') -> Union[CommandResults, str]:
     """
-    https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-user-search/#api-rest-api-3-user-search-get
+    https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-user-search/#api-rest-api-3-user-picker-get
```

Comment:
no need to change. you're still using the user/search endpoint and not the picker one.